src/index/gpu_raft/gpu_raft_ivf_pq_config.h

```diff
@@ -46,7 +46,7 @@ struct GpuRaftIvfPqConfig : public IvfPqConfig {
             .description("search for top k similar vector.")
             .set_range(1, 1024)  // Declared in base but limited to 1024
             .for_search();
-        KNOWHERE_CONFIG_DECLARE_FIELD(m).set_default(8).description("m").set_range(1, 65536).for_train();
+        KNOWHERE_CONFIG_DECLARE_FIELD(m).set_default(0).description("m").set_range(0, 65536).for_train();
```

Comment:
If m is set to 0, does it mean auto? We ran some tests here, and setting m to 0 caused some issues.